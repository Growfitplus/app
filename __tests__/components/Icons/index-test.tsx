import {
  ArrowBackIcon,
  ArrowNextIcon,
  CameraIcon,
  ChevronNextIcon,
  LogOutIcon,
} from '@/components/Icons';
import { render } from '@testing-library/react-native';

describe('Icons', () => {
  it('should render ArrowBackIcon component', () => {
    const { toJSON } = render(<ArrowBackIcon />);

    expect(toJSON).toMatchSnapshot();
  });

  it('should render ArrowNextIcon component', () => {
    const { toJSON } = render(<ArrowNextIcon />);

    expect(toJSON).toMatchSnapshot();
  });

  it('should render ChevronNextIcon component', () => {
    const { toJSON } = render(<ChevronNextIcon />);

    expect(toJSON).toMatchSnapshot();
  });

  it('should render LogOutIcon component', () => {
    const { toJSON } = render(<LogOutIcon />);

    expect(toJSON).toMatchSnapshot();
  });

  it('should render CameraIcon component', () => {
    const { toJSON } = render(<CameraIcon />);

    expect(toJSON).toMatchSnapshot();
  });
});
