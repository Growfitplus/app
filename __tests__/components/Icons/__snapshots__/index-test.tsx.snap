// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons should render ArrowBackIcon component 1`] = `[Function]`;

exports[`Icons should render ArrowNextIcon component 1`] = `[Function]`;

exports[`Icons should render CameraIcon component 1`] = `[Function]`;

exports[`Icons should render ChevronNextIcon component 1`] = `[Function]`;

exports[`Icons should render LogOutIcon component 1`] = `[Function]`;
